src/hooks/useVideoGridRows.tsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from 'react'
+
+import { useMediaMatch } from '@/hooks/useMediaMatch'
+
+type UseVideoGridRowsArgs = 'videos' | 'videosInChannel'
```

Comment:
How about this?
```suggestion
type VideoGridRowsVariant = 'default' | 'compact'
```

Reply:
I'm also wondering whether we should use the exact same policy for channel view - we previously used 4 rows by default there, now it will use 2 on biggest screen size, like content sections. Waiting for reply from Kuba as to what he thinks we should do there

Reply:
> How about this?

I wanted to use more generic names but I wasn't sure if we could actually call it a default, especially if we get one more variant.

Reply:
Yeah let's wait for now

Reply:
Okay, let's have a separate variant for channel view in which we will use 4 rows on the biggest screens. You can also adjust other breakpoints for that as you see fit